src/components/shared/DlTooltip/DlTooltip.vue

```diff
@@ -360,7 +358,7 @@ export default defineComponent({
 
         function configureScrollTarget() {
             if (anchorEl.value !== null || props.scrollTarget) {
-                (localScrollTarget as Ref<any>).value = getScrollTarget(
+                ;(localScrollTarget as Ref<any>).value = getScrollTarget(
```

Comment:
what does `;` do?

Reply:
@makc-dataloop This is an automatic change.